lib/cocoapods/generator/copy_resources_script.rb

```diff
@@ -76,7 +76,7 @@ def script
           unless resources.empty?
             script += %(if [[ "$CONFIGURATION" == "#{config}" ]]; then\n)
             resources.each do |resource|
-              script += "  install_resource '#{resource}'\n"
+              script += "  install_resource \"#{resource}\"\n"
```

Comment:
@segiddins Could you clarify what should be escaped here. As discussed above there are instances where `resource` contains expressions intended for expansion later.